app/routers/event.py

```diff
@@ -52,3 +52,77 @@ def sort_by_date(events: List[Event]) -> List[Event]:
 
     temp = events.copy()
     return sorted(temp, key=attrgetter('start'))
+
+
+def check_validation(start_time, end_time) -> bool:
+    """Check if the start_date is smaller then the end_time"""
+
+    try:
+        return start_time < end_time
+    except TypeError:
+        return False
+
+
+def add_new_event(values: dict, db) -> Event:
+    """Get User values and the DB Session insert the values
+    to the DB and refresh it exception in case that the keys
+    in the dict is not match to the fields in the DB
+    return the Event Class item"""
+
+    try:
+        if check_validation(values['start'], values['end']):
+            new_event = create_model(
```

Comment:
In order to retrieve all participants of an event, we need to query the "UserEvent" association table. because you didn't add the owner to the association table, it will look like he isn't invited to the event. consider adding this change.

Reply:
tanks!